src/extension.ts

```diff
@@ -20,13 +20,17 @@ interface Rangeable {
 }
 
 export function activate(context: ExtensionContext) {
-    const eventDisposable = (workspace as any).onWillSaveTextDocument(e => {
+    const eventDisposable = workspace.onWillSaveTextDocument(e => {
```

Comment:
I've updated the "engines": "vscode" to 1.8.0, where onWillSaveTextDocument is defined on workspace. You probably need to run `npm install` before compilation.